### PR TITLE
CM-285: null check for ckeditor component

### PR DESCRIPTION
**Pull request by foley-lynn**
### Jira Ticket:
CM-285

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/CM-285

### Description:
CKeditor was throwing an error if there was null content in a field from import. Added null check.

### CODE REVIEW

File: src/cms/plugins/wysiwyg/admin/src/components/CKEditor/index.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ const Wrapper = styled.div`
 	.ck-dropdown .ck-button.ck-dropdown__button .ck-button__label {
 		width: 6em;
 	}
-	.ck-editor__editable {
-		min-height: 200px;
+	.ck-editor__main > div {
+		min-height: 200px
 	}
```
Comment (ayumi-oxd) [inline src/cms/plugins/wysiwyg/admin/src/components/CKEditor/index.js:12]: NIT: You forgot `;` 😄

**Review by ayumi-oxd — COMMENTED** [review]
Question: I tested on my local but I'm still getting a similar error. Could it be a different error from this ticket? Sorry if I miss something. Tested on Chrome.

```
main.540a67c2.chunk.js:1 TypeError: Cannot convert undefined or null to object at Function.keys (<anonymous>)  at n.value (main.540a67c2.chunk.js:1:10554783)  at n.<anonymous> (main.540a67c2.chunk.js:1:10260916)  at n.value (main.540a67c2.chunk.js:1:10256336)  at <computed> [as set] (main.540a67c2.chunk.js:1:10261033)  at n.value (main.540a67c2.chunk.js:1:10708877)  at Object.onReady (main.540a67c2.chunk.js:1:9876879)  at main.540a67c2.chunk.js:1:9905123{phase: 'runtime', willEditorRestart: true}
```
Screen Shot 2022-12-30 at 10 55 32 AM (2)

File: src/cms/plugins/wysiwyg/admin/src/components/CKEditor/index.js (hunk 1 above)
Comment (foley-lynn) [reply src/cms/plugins/wysiwyg/admin/src/components/CKEditor/index.js:12]: Love it. Thanks. :-)

**Review by ayumi-oxd — APPROVED** [review]
After I run `npm run build && npm run develop`, it works as expected. I don't see any errors.

**Review by molund — APPROVED** [review]

**Review by nataliasabadysh — APPROVED** [review]

**Pull request merged**